pkg/controller/release/release_controller_test.go

```diff
@@ -927,7 +939,7 @@ func (f *fixture) expectAssociatedObjectsCreated(release *shipper.Release, clust
 			"Normal ReleaseScheduled Created CapacityTarget \"%s\"",
 			relKey,
 		),
-	}
+	}...)
```

Comment:
Bit of a nitpick, but the ellipsis here are a bit weird, as we're building a `[]string` just to unpack it immediately.